examples/16-cancel-order-line.py

```diff
@@ -0,0 +1,53 @@
+# coding=utf-8
+#
+# Example 16 - Cancel an order line using the Mollie API.
+#
+from __future__ import print_function
+
+import os
+
+from mollie.api.client import Client
+from mollie.api.error import Error
+
+
+def main():
+    try:
+        #
+        # List the most recent orders
+        #
+        # See: https://docs.mollie.com/reference/v2/orders-api/list-orders
+        #
+        api_key = os.environ.get('MOLLIE_API_KEY', 'test_test')
+        mollie_client = Client()
+        mollie_client.set_api_key(api_key)
+
+        #
+        # Cancel an order line with ID "odl_dgtxyl" for order ID "ord_8wmqcHMN4U"
+        #
+        # See: https://docs.mollie.com/reference/v2/orders-api/cancel-order-line
+        #
+
+        order_id = 'ord_8wmqcHMN4U'
```

Comment:
This can never work for random persons using their own API key.
Try the 'fetch a list of orders and use the first' approach from other example files.